x-pack/solutions/security/plugins/cloud_security_posture/public/assets/icons/nvd_logo_svg.tsx

```diff
@@ -5,24 +5,28 @@
-    />
-    <path
-      fill={euiThemeVars.euiColorFullShade}
-      d="M 22.050781 10.714844 C 22.941406 10.605469 23.636719 10.164062 24.042969 9.285156 C 25.1875 7.234375 26.230469 5.117188 27.308594 3.027344 C 27.472656 2.707031 27.691406 2.558594 28.058594 2.5625 C 30.578125 2.574219 33.097656 2.558594 35.617188 2.574219 C 36.417969 2.578125 37.136719 2.894531 37.6875 3.488281 C 38.433594 4.296875 38.667969 5.28125 38.277344 6.324219 C 37.851562 7.460938 36.976562 8.109375 35.800781 8.152344 C 34.171875 8.214844 32.539062 8.171875 30.90625 8.167969 C 30.464844 8.167969 30.34375 8.03125 30.339844 7.585938 C 30.339844 6.332031 30.335938 5.074219 30.347656 3.820312 C 30.347656 3.554688 30.28125 3.449219 30.003906 3.457031 C 29.382812 3.476562 28.757812 3.480469 28.136719 3.457031 C 27.84375 3.449219 27.785156 3.566406 27.785156 3.839844 C 27.792969 6.003906 27.796875 8.167969 27.78125 10.332031 C 27.777344 10.660156 27.886719 10.730469 28.183594 10.730469 C 30.210938 10.722656 32.238281 10.722656 34.265625 10.722656 C 34.972656 10.722656 35.738281 10.824219 36.445312 10.71875 C 37.734375 10.519531 38.828125 9.960938 39.699219 8.945312 C 40.890625 7.5625 41.203125 5.957031 40.878906 4.195312 C 40.667969 3.046875 40.101562 2.089844 39.242188 1.3125 C 38.371094 0.527344 37.359375 0.0390625 36.207031 0.0273438 C 33.074219 -0.0117188 29.945312 0.015625 26.8125 0.0117188 C 26.296875 0.0117188 25.921875 0.261719 25.671875 0.714844 C 24.382812 3.046875 23.355469 5.570312 21.941406 7.828125 C 21.765625 8.105469 21.605469 8.109375 21.433594 7.84375 C 19.898438 5.476562 18.742188 2.777344 17.414062 0.289062 C 17.3125 0.101562 17.183594 0.00390625 16.953125 0.0078125 C 16.480469 0.0195312 16.007812 0.0195312 15.535156 0.015625 C 15.265625 0.015625 14.988281 0.0351562 14.714844 0.0117188 C 14.640625 0.0078125 14.5625 0 14.5 0.0664062 C 14.484375 0.148438 14.519531 0.210938 14.554688 0.273438 C 14.839844 0.785156 15.097656 1.3125 15.371094 1.832031 C 16.171875 3.367188 16.96875 4.90625 17.757812 6.445312 C 18.234375 7.367188 18.707031 8.285156 19.179688 9.203125 C 19.359375 9.542969 19.582031 9.847656 19.882812 10.101562 C 20.519531 10.644531 21.257812 10.8125 22.050781 10.714844 Z M 22.050781 10.714844 "
-    />
-  </svg>
-);
+export const NvdLogo = () => {
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="42px"
+      height="10px"
+      viewBox="0 0 41 11"
+      version="1.1"
+    >
+      <path
+        fill={euiTheme.colors.fullShade}
```

Comment:
shade colors are recommended to be replaced too, but there is no clear guidance yet on what to replace them with.